packages/dds/tree/src/test/shared-tree/treeCheckout.spec.ts

```diff
@@ -932,8 +932,7 @@ describe("sharedTreeView", () => {
 		assert.equal(checkout2Revertibles.redoStack.length, 1);
 		assert.equal(view2.checkout.getRemovedRoots().length, 2);
 
-		const sf2 = new SchemaFactory("schema2");
-		provider.trees[0].kernel.checkout.updateSchema(toStoredSchema(sf2.array(sf1.number)));
+		provider.trees[0].kernel.checkout.updateSchema(toStoredSchema([sf1.number, schema1]));
```

Comment:
This schema change made the document out of schema (since the structurally named array gets a different name with different allowed types, resulting in a document root with a type thats not in the schema anymore). The fixed version does an actually valid schema upgrade, from the old array to a union of number and the old array.